web/embed_style.html

```diff
@@ -28,6 +28,14 @@
     --paper-toggle-button-unchecked-ink-color: var(--text-primary-color);
   }
   
+  .default-text-div {
+    width: 100%;
+    height: 100%;
```

Comment:
I'll note this for future updates. Merging this in now in order to proceed with vulcanization changes.